apps/api/src/app/widgets/dtos/user-preference.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsBoolean, IsOptional, ValidateNested } from 'class-validator';
+import { ChannelTypeEnum } from '@novu/stateless';
```

Comment:
```suggestion
import { ChannelTypeEnum } from '@novu/shared';
```
as stateless does not have all types 
@djabarovgeorge

Reply:
I do not like the one in shared, I think we need to refactor it.
what do you think?

Reply:
Why don't you like it?

Reply:
because there are other types then channels